### ISSUE TITLE
ATXBuldingConnect

### ISSUE THREAD

**Issue by BuffaloCloud** [atxhack4change/2016-project-proposals#10]
## Short description of the idea (140 character tweet):

An app that lets you inform ATX and private developers what you want in your ‘hood and gives two way communication for planned development btwn residents and City

## What is your project concept or idea? What challenge or opportunity will it solve?

The app will allow residents to inform City departments what types of development they want in their neighborhood. The app will enable two-way communication. City departments will solicit community input on current projects and will be able to determine if there is support for them. Resident will be able to notify City departments of unwanted development in their neighborhoods.

## Who will benefit from your project? Describe the humans at the center of the problem - who are they? 

Residents, especially in East Austin, Rundburg, and other under represented areas will have a way of having their voices heard when it comes to developments in their neighborhoods.

### How would they benefit? Can you tell their story?

Community feedback has been amorphous. It needs to be targeted to the right department for the right reasons. I can tell their story especially folks in East Austin.

### Do you have photos of them? Or quotes?

I can get them.

### [Look at the right sidebar :arrow_upper_right: ! Do you see the Labels? Pick a few]

- Economic Development
- Economic Opportunity
- Environment
- Housing
- Mobility/Transportation
- Neighborhoods
- Open Government/Transparency

## What is the current state of your project idea? [Pick more labels :arrow_upper_right: ]

- I have an idea or **concept** of a solution, but have not yet prototyped or tested it

## Tell us more about the current state of your project idea.

I've discussed the idea with 20 East Austin residents and they are supportive of it.

### Do you have any links to text, mockups, code, or data?

I do not.

## Tell us about yourself. 

I am a small business owner who lives in District 1.

### What is your background? Why is this challenge important to you? 

I do not like seeing people priced out of their homes because the taxes triple in a year just because of gentrification. Current residents should be able to have a say in the type of homes, industries, and "improvements" that are made in their neighborhoods.

### Are you representing a group in submitting this project idea? 

No.

**Comment by mateoclarke** [comment]
Hi @BuffaloCloud. Did you mean to close out this proposal? Seems like a good project. Feel free to email me at info@open-austin.org if you have an concerns you want to discuss.

**Comment by BuffaloCloud** [comment]
Nope. I did not mean to close it.I was not sure what I was doing.Thanks
 Christine MosesBuffalo Cloud Consulting LLCPO Box 144132Austin, TX 78714Phone: 512-215-4436Cell: 860-803-5511BuffaloCloudConsulting.comBuffalocloudconsulting@yahoo.comTexas Certified Historically Underutilized Business (HUB)Austin MBE and WBE, U.S. DOT DBE Certified

**Comment by mateoclarke** [comment]
Thanks @BuffaloCloud, I've added some labels based on your answer.

**Comment by BuffaloCloud** [comment]
THANK YOU Christine MosesBuffalo Cloud Consulting LLCPO Box 144132Austin, TX 78714Phone: 512-215-4436Cell: 860-803-5511BuffaloCloudConsulting.comBuffalocloudconsulting@yahoo.comTexas Certified Historically Underutilized Business (HUB)Austin MBE and WBE, U.S. DOT DBE Certified

**Comment by ssharif1** [comment]
### ACTION NEEDED 

Hello Project Champion, 

Please provide the following:
Full Name + Email Address 
You can comment here or send to ssharif1@stedwards.edu 

We are going to be sending out a note + newsletter to our 345 (as of May 18) attendees sharing our submitted projects before the event and your hackers will want to know who you are. Additionally, we have PC-specific communication that we want to make sure that you are included on.

Thanks,
ATX H4C Staff

**Comment by mateoclarke** [comment]
Hi @BuffaloCloud! You're project has been accepted!

Now that your project has been approved, here are some things we suggest you do next:
https://github.com/atxhack4change/2016-project-proposals#once-your-project-is-accepted-here-are-some-things-we-suggest-you-do-next

You will receive an email in the next few days asking you to confirming your speaking slot for the Saturday morning pitches. 

Please let us know if you will no longer be able to champion your project at the hackathon.

**Comment by BuffaloCloud** [comment]
Wow! This is mighty cool.Thanks for letting me know.I will be there to champion it.Thank you!Chris Christine MosesBuffalo Cloud Consulting LLCPO Box 144132Austin, TX 78714Phone: 512-215-4436Cell: 860-803-5511BuffaloCloudConsulting.comBuffalocloudconsulting@yahoo.comTexas Certified Historically Underutilized Business (HUB)Austin MBE and WBE, U.S. DOT DBE Certified

**Comment by mateoclarke** [comment]
You could also be looking into what data the city provides on upcoming developments. 

Here are some leads for you to explore:
https://data.austintexas.gov/browse?category=Capital+Planning&q=development&utf8=%E2%9C%93
https://data.austintexas.gov/browse?q=land+use&sortBy=relevance&utf8=%E2%9C%93

**Comment by BuffaloCloud** [comment]
Yes!I found the pending projects site the other day when I presented at the City's Open Data meeting. I found it while I was sitting at the meeting. I will be using it as a starting point. There may not be a lot to do other than to notify people of its existence and somehow connect the community response to the right department. All good.
Thank you so much for your guidance. I'll be asking you questions over the weekend. I'll work on this on Monday.
Chris Christine MosesBuffalo Cloud Consulting LLCPO Box 144132Austin, TX 78714Phone: 512-215-4436Cell: 860-803-5511BuffaloCloudConsulting.comBuffalocloudconsulting@yahoo.comTexas Certified Historically Underutilized Business (HUB)Austin MBE and WBE, U.S. DOT DBE Certified

**Comment by danielhonker** [comment]
@BuffaloCloud - you should have received an invite to join Slack. If you haven't already joined the team and seen the Slack channel for your project, it's #atxbuildingconnect. We'll be sending out an email with details on pitches soon.
